src/main/java/seedu/foodrem/logic/commands/generalcommands/ExitCommand.java

```diff
@@ -13,8 +13,37 @@ public class ExitCommand extends Command {
     private static final String MESSAGE_SUCCESS = "Exiting FoodRem as requested ...";
 
     @Override
-    public CommandResult execute(Model model) {
-        return new CommandResult(MESSAGE_SUCCESS, false, true);
```

Comment:
Do you think it's necessary to add a comment here explaining why you used an anonymous class? Ditto for Help, Reset commands

Reply:
For the anonymous class itself? I don't think so, it's pretty standard Java.